modules/build/src/test/scala/scala/build/tests/SourcesTests.scala

```diff
@@ -501,4 +506,132 @@ class SourcesTests extends munit.FunSuite {
     }
   }
 
+  test("throw error when exclude found in multiple file") {
```

Comment:
I think we should also test error if exclude comes from outside `project.scala` and from `project.scala` that is not top-level